nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -631,39 +570,6 @@ export default class NasesService {
     }
   }
 
-  async canSendForm(
-    formId: string,
-    user: JwtNasesPayloadDto,
-    userExternalId?: string,
-  ): Promise<boolean> {
-    const form = await this.formsService.checkFormBeforeSending(formId)
-    if (
-      !this.formsHelper.userCanSendFormEid(
-        form,
-        user.sub,
-        user.actor.sub,
-        userExternalId,
-      )
-    ) {
-      throw this.throwerErrorGuard.ForbiddenException(
-        NasesErrorsEnum.FORBIDDEN_SEND,
-        NasesErrorsResponseEnum.FORBIDDEN_SEND,
-      )
-    }
-
-    const formAttachmentsReady =
```

Comment:
This is checked twice in the process:
https://github.com/bratislava/konto.bratislava.sk/blob/9e3395a0a39e6ae9d1c414504af7de0662a55365/nest-forms-backend/src/nases/nases.service.ts#L505